app/calculator.js

```
@@ -0,0 +1,51 @@
+/* eslint-disable linebreak-style */
+/**
+ *
+ *A simple class containing two methods
+ * @class Calculator
+ */
+class Calculator {
+  /**
+   * Creates an instance of Calculator.
+   * @memberof Calculator
+   */
+  constructor() {}
+
+  /**
+   *
+   *
+   * @param {*} args array of numbers
+   * @param {*} elementType the value of checking the array for a number
+   * @param {*} element each element of the array
+   * @return {*} add all numbers
+   * @memberof Calculator
+   */
+  add(...args) {
+    let elementType = args.every((element) => typeof element === 'number');
+    if (elementType === true) {
+      return args.reduce((acc, curr) => acc + curr);
+    } else {
+      return 'Variable in an array is not a number';
```

You should throw an error if one or more parameters are not of the `number` type and then check the Error message if a user tries to pass such parameter.